### DISCUSSION TITLE
DateRangePicker, can I let the 2nd Calendar's date stay unchanged when I selected the date of the first Calendar?

### DISCUSSION THREAD

**Discussion by helloint** [rsuite/rsuite#3493] [General]
Currently, if I selected the date from the first Calendar, the 2nd Calendar will automatically switch to the next month of the first calendar. I hope I can let the 2nd calendar unchanged.
I checked the props, and didn't see such flag. I tried to check the source code but still no luck.
Thanks.

**Comment by simonguo** [comment]
Not supported yet
